dev/make-distribution.sh

```diff
@@ -123,26 +123,26 @@ fi
 if [ ! "$(command -v "$MVN")" ] ; then
     echo -e "Could not locate Maven command: '$MVN'."
     echo -e "Specify the Maven command with the --mvn flag"
-    exit -1;
+    exit 255;
 fi
 
-VERSION=$("$MVN" help:evaluate -Dexpression=project.version $@ \
+VERSION=$("$MVN" help:evaluate -Dexpression=project.version "$@" \
```

Comment:
SC2068 (error): Double quote array expansions to avoid re-splitting elements.

Reply:
Same, this doesn't seem to apply